sdk/state/ingest.go

```diff
@@ -14,6 +15,7 @@ import (
 // TODO: Accept the xdr.TransactionResult and xdr.TransactionMeta so code can
 // determine if successful or not, and understand changes in the ledger as a
 // result.
+// TODO: Make sure the ingested tx is fee bumped.
```

Comment:
this was a reminder not necessarily for this PR but in general making sure the submitted transaction to the network is fee bumped, otherwise the balances will be off if the escrow account is going to pay or has paid a transaction fee.

Maybe the TODO should be moved somewhere else as it may not directly apply to `IngestTx` 🤔

Reply:
Now that you're switching to the result meta xdr approach this shouldn't matter since a non-fee bumped tx that reduces the balance of the account should result in the new balance being less? I think anyway.

The txbuild package only builds txs with zero fees, so other code is required to fee bump the txs when submitting. I think validating that is probably out of scope, or not needed.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/234/commits/9d8a554876535a0aa13a2d99fc7f33692c096ba3